modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_metadata/metadata_entry.jsp

```diff
@@ -87,7 +87,13 @@ else if (metadataField.equals("view-count")) {
 	<c:when test='<%= Objects.equals(value, "author") %>'>
 
 		<%
-		User assetRendererUser = UserLocalServiceUtil.getUser(assetRenderer.getUserId());
+		User assetRendererUser = null;
+
+		try {
+			assetRendererUser = UserLocalServiceUtil.getUser(assetRenderer.getUserId());
```

Comment:
If we will do nothing on the exception.
It not better idea to use the `fetch` methods?